Packs/Code42/ReleaseNotes/6_0_0.md

```diff
@@ -0,0 +1,49 @@
+
+#### Integrations
+
+##### Code42 Event Collector
+
+- Updated the Docker image to: *demisto/py42:1.0.0.117258*.
+
+##### Code42
+- Updated the Docker image to: *demisto/py42:1.0.0.117258*.
+
+- Added API Gateway URL configuration field for the Code42 integration. This is necessary to support alert sessions.
+- Updated incident fetch to use alert sessions instead of legacy alerts.
```

Comment:
Please ensure that each line in the release notes starts with one of the approved prefixes.
You can refer to the list of acceptable prefixes in the following link:
[Release Notes Guidelines](https://xsoar.pan.dev/docs/documentation/release-notes#general)

Reply:
I've updated here. Incidentally, the guidelines do not include any line prefixes for "removed deprecated commands", so I had to format those lines as "Fixed an issue where commands that used deprecated Code42 APIs could still be called". Is it possible to add a line prefix for removing commands that no longer should be in?